packages/blocks/src/database-block/utils.ts

```diff
@@ -61,3 +70,95 @@ export const getTagColor = tagColorHelper();
 export function isDivider(action: DatabaseAction): action is Divider {
   return action.type === 'divider';
 }
+
+type InitLimitedLengthVEditorConfig = {
+  yText: Y.Text | string;
+  container: HTMLElement;
+  targetModel: BaseBlockModel;
+  maxLength: number;
+  readonly?: boolean;
+  handlers?: VirgoEventService<BaseTextAttributes>['_handlers'];
+  options?: VEditorOptions;
+};
+export function initLimitedLengthVEditor({
```

Comment:
It doesn't feel right requiring so much setup and handling events like `compositionend` to implement a length limit. How about supporting this as virgo configuration @Flrande ?